lib/puppet/provider/package/tap.rb

```diff
@@ -37,7 +37,7 @@ def query
       output = execute([command(:brew), :tap])
       output.each_line do |line|
         line.chomp!
-        return { :name => line, :ensure => 'present', :provider => 'tap' } if line.downcase == name
+        return { :name => line, :ensure => 'present', :provider => 'tap' }  if line.downcase == name || line.downcase == name.gsub('homebrew-', '')
```

Comment:
Could you just do this as `[name, name.gsub('homebrew-', '')].include? line.downcase`? I think that'll be a bit easier to parse, in case we ever have to add more cases.